ocp-templates/mpp/backend.yaml

```diff
@@ -25,7 +25,7 @@ objects:
     template:
       metadata:
         labels:
-          app: ibutsu-backend
+          app: ${APP_NAME}
```

Comment:
This is going to break the Service if the APP_NAME is not set correctly, the APP_NAME's default value is `ibutsu-server` - which is also used by all the other templates (with the same value), while the Service's label selector should use a unique value (otherwise more than one pod will match "app=ibutsu-server"